src/flows/CostCalculator/CostCalculatorForm.tsx

```diff
@@ -30,13 +30,19 @@ type CostCalculatorFormProps = Partial<{
    * Whether to reset the form when the form is successfully submitted.
    */
   shouldResetForm?: boolean;
+
+  /**
+   * Fields to reset when the form is successfully submitted.
+   */
+  shouldResetFormFields?: string[];
 }>;
 
 export function CostCalculatorForm({
   onSubmit,
   onError,
   onSuccess,
   shouldResetForm,
+  shouldResetFormFields,
```

Comment:
We should find a better name, because it sound like it's a boolean, when in fact it's a list. Maybe `resetFields`?